multistateview/src/main/java/me/xdj/view/MultiStateView.java

```diff
@@ -142,8 +147,10 @@ public Component getView(int state) {
      */
     public Component getCurrentView() {
         Component view = getView(mCurrentState);
-        if (view == null && mCurrentState != STATE_CONTENT) {
-            throw new NullPointerException("current state view is null, state = " + mCurrentState);
+        if (view == null && mCurrentState == STATE_CONTENT) {
+            Logger.getLogger("Content is null");
```

Comment:
if (view == null && mCurrentState == STATE_CONTENT) -- the code flow continues inspite of encountering a null condition. In ideal case, the flow must terminate here. Please check & understand the base library implementation